dev/bots/analyze-sample-code.dart

```diff
@@ -285,14 +285,14 @@ class SampleChecker {
         <String>[
           '--snapshot=$_snippetsSnapshotPath',
           '--snapshot-kind=app-jit',
-          path.canonicalize(_snippetsExecutable),
-        ]..addAll(args),
+          path.canonicalize(_snippetsExecutable), ...args,
```

Comment:
This is directly after the quick assist runs?

I know this code doesn't get dartfmt'd, but I wonder if dartfmt would move the `...args` to the next line? I think that would make this construct slightly easier to read.

Reply:
It would, yes.